__tests__/app.test.js

```diff
@@ -75,6 +75,93 @@ describe('GET /api/articles', () => {
+        return request(app)
+        .get('/api/articles?sortOn=title&order=ASC')
+        .expect(200)
+        .then(({ body }) => {
+            expect(Array.isArray(body)).toBe(true);
+            expect(body).toHaveLength(12);
+            expect(body).toBeSortedBy('title', {
+                ascending: true, 
+            })
+            body.forEach((article) => {
+                expect(article.author).toEqual(expect.any(String)); 
+                expect(article.title).toEqual(expect.any(String)); 
+                expect(article.article_id).toEqual(expect.any(Number));
+                expect(article.topic).toEqual(expect.any(String));
+                expect(article.created_at).toEqual(expect.any(String));
+                expect(article.votes).toEqual(expect.any(Number));
+                expect(article.comment_count).toEqual(expect.any(Number)); 
+            })
+        })
+    });
+    it('Status: 200 and an array of object sorted by comment_count in descending order when given queries ', () => {
+        return request(app)
+        .get('/api/articles?sortOn=comment_count&order=DESC')
+        .expect(200)
+        .then(({ body }) => {
+            expect(Array.isArray(body)).toBe(true);
+            expect(body).toHaveLength(12);
+            expect(body).toBeSortedBy('comment_count', {
+                descending: true,
+            })
+            body.forEach((article) => {
+                expect(article.author).toEqual(expect.any(String)); 
+                expect(article.title).toEqual(expect.any(String)); 
+                expect(article.article_id).toEqual(expect.any(Number));
+                expect(article.topic).toEqual(expect.any(String));
+                expect(article.created_at).toEqual(expect.any(String));
+                expect(article.votes).toEqual(expect.any(Number));
+                expect(article.comment_count).toEqual(expect.any(Number)); 
+            })
+        })
+    });
+    it('Status: 200 and an array of objects filtered by the input topic', () => {
+        return request(app)
+        .get('/api/articles?term=mitch&sortOn=votes&order=ASC')
+        .expect(200)
+        .then(({ body }) => {
+            expect(Array.isArray(body)).toBe(true);
+            expect(body).toHaveLength(11);
+            expect(body).toBeSortedBy('votes', {
+                ascending: true, 
+            })
+            body.forEach((article) => {
+                expect(article.author).toEqual(expect.any(String)); 
+                expect(article.title).toEqual(expect.any(String)); 
+                expect(article.article_id).toEqual(expect.any(Number));
+                expect(article.topic).toEqual('mitch');
+                expect(article.created_at).toEqual(expect.any(String));
+                expect(article.votes).toEqual(expect.any(Number));
+                expect(article.comment_count).toEqual(expect.any(Number)); 
+            })
+        })
```

Comment:
Like I said previously, if you're testing for query and have already tested for the shape of the objects previously, you don't need to check them again each time.